aptos-node/src/lib.rs

```diff
@@ -301,14 +328,74 @@ where
     start(config, Some(log_file), false)
 }
 
+/// Merge two YAML values, useful for merging NodeConfigs
+fn merge_yaml(a: &mut serde_yaml::Value, b: serde_yaml::Value) {
```

Comment:
You can do this or you can add serde_merge

https://docs.rs/serde_merge/latest/serde_merge/

Reply:
ah yea that sounds like a better idea